src/control/Control.Layers.js

```diff
@@ -198,6 +198,7 @@ export var Layers = Control.extend({
 		var link = this._layersLink = DomUtil.create('a', className + '-toggle', container);
 		link.href = '#';
 		link.title = 'Layers';
+		link.setAttribute('aria-label', 'Toggle Layer Contol');

```

Comment:
- [ ] Typo: missing "r" in "control". Additionally I think aria-labels and titles in other places aren't capitalized.
- [ ] Please add `role="button"` such that screen readers don't announce this as a regular link.

❤️

Reply:
Thanks for the suggestions @Malvoz  - changes made